.travis.yml

```diff
@@ -16,6 +16,7 @@ install:
     - pip install -U pip setuptools
     - pip install -U coverage coveralls
     - pip install -U -e .[test,docs]
+    - pip install -e git+https://github.com/minddistrict/zope.i18nmessageid.git@CB-556-FR-translation-with-C#egg=zope.i18nmessageid
```

Comment:
We can drop this now; zope.i18nmessageid 4.3 is released to PyPI.

(But please add a version constraint to `setup.py`.)